starter_code/src/movies.js

```diff
@@ -1,19 +1,111 @@
-/* eslint no-restricted-globals: 'off' */
+/*eslint no-restricted-globals: 'off'*/
+
 
 // Iteration 1: All rates average - Get the average of all rates with 2 decimals 
 
- 
+function ratesAverage(movies) {
+    let arrayOfRates = movies.map(function (movies) {
+        let rate = parseFloat(movies.rate);
```

Comment:
while this is good and works, you could be doing the same inside the `reduce` function, to avoid iterating twice over the array 👍